features/mbedtls/platform/inc/platform_mbed.h

```diff
@@ -17,6 +17,24 @@
  *  This file is part of mbed TLS (https://tls.mbed.org)
  */
 
+
+#ifndef __PLATFORM_MBED__H__
+#define __PLATFORM_MBED__H__
+
+#if (defined(TARGET_PSA) && defined(MBEDTLS_ENTROPY_NV_SEED))
+
+#include "default_random_seed.h"
```

Comment:
Is this possible to be in target PSA with NV_SEED but in the non-secure side? 
If yes, we will have compilation error on default_random_seed.h file would not found

Reply:
Thre are two use cases:
1. PSA target (with SPE and NSPE) and we are building for NSPE. e.g. `FUTURE_SEQUANA_PSA` for such a targets it is not expected to have `MBEDTLS_ENTROPY_NV_SEED` macro enabled.
2. PSA compliant target (NSPE only). e.g. K64F. IIRC you have tested it on such a target yourself :)